interp/declare.ml

```diff
@@ -384,15 +384,16 @@ let inInductive : inductive_obj -> obj =
 
 let declare_projections univs mind =
```

Comment:
It's a bit strange to be declaring projections here instead of in the Record.declare_projections. Maybe another PR should move this code.

Reply:
Beware that `comInductive` also needs to declare projections sometimes (I think?).